doc/api/https.md

```diff
@@ -167,9 +167,10 @@ changes:
 
 Makes a request to a secure web server.
 
-The following additional `options` from [`tls.connect()`][] are also accepted
-when using a custom [`Agent`][]: `ca`, `cert`, `ciphers`, `clientCertEngine`,
-`key`, `passphrase`, `pfx`, `rejectUnauthorized`, `secureProtocol`, `servername`
+The following additional `options` from [`tls.connect()`][] are also accepted:
+`pfx`, `key`, `passphrase`, `cert`, `ca`, `ciphers`, `clientCertEngine`, `crl`,
+`dhparam`, `ecdhCurve`, `honorCipherOrder`,`rejectUnauthorized`, `secureOptions`,
+`secureProtocol`, `servername`, `sessionIdContext`
```

Comment:
The entries were originally sorted by alphabet. Any reason why you changed that?

Reply:
I don't remember exactly but I believe it was to align with the list in [`tls.createSecureContext()`](https://nodejs.org/api/tls.html#tls_tls_createsecurecontext_options). That being said, alphabetic probably makes more sense.